### PR TITLE
refactor: move log config template to emqx_conf app

### PR DESCRIPTION
**Pull request by zmstone**
Release version: v/e5.8.3

## Summary

Move the `log` config template from `emqx` app to `emqx_conf` app.
This is because the `log` config schema is defined in `emqx_conf` app.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [~] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [~] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [x] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: apps/emqx_conf/etc/emqx_conf.conf [hunk 1]
```diff
@@ -19,3 +19,23 @@ cluster {
...
+## Read more about configs here: {{ emqx_configuration_doc_log }}
+
+#log {
+    # file {
+    #     level = warning
+    # }
+    # console {
+    #     level = warning
+    # }
+#}
```
Comment (keynslug) [inline apps/emqx_conf/etc/emqx_conf.conf:41]:
```suggestion
# log {
#     file {
#         level = warning
#     }
#     console {
#         level = warning
#     }
# }
```

File: .ci/docker-compose-file/docker-compose-redis-single-tcp.yaml [hunk 2]
```diff
@@ -6,8 +6,8 @@ services:
     image: public.ecr.aws/docker/library/redis:${REDIS_TAG}
     volumes:
       - ./redis/single-tcp:/usr/local/etc/redis/
-    ports:
-      - "6379:6379"
+    #ports:
+      #- "6379:6379"
```
Comment (keynslug) [inline .ci/docker-compose-file/docker-compose-redis-single-tcp.yaml:10]: Leftovers?

Comment (zmstone) [reply .ci/docker-compose-file/docker-compose-redis-single-tcp.yaml:10]: no, this is causing test failures in master branch

**Review by keynslug — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-5089)

**Pull request merged**